src/SourceBuild/content/repo-projects/runtime.proj

```diff
@@ -1,6 +1,10 @@
 <Project>
   <Import Project="Sdk.props" Sdk="Microsoft.NET.Sdk" />
 
+  <PropertyGroup>
```

Comment:
IMO, this should be grouped with the build arguments as it arguably is one.  It controls the dependency flow used during the build.